cli/tests/integration_tests.rs

```diff
@@ -426,6 +426,54 @@ fn cache_invalidation_test_no_check() {
   assert_eq!(actual, "43\n");
 }
 
+#[tokio::test]
+async fn local_sources_not_cached_in_memory() {
+  let deno_dir = TempDir::new().expect("tempdir fail");
+  let fixture_a_path = deno_dir.path().join("fixture_a.js");
+  let fixture_a = r#"
+    const worker_a = new Worker(
+      new URL("fixture_b.ts", import.meta.url).href,
+      { type: "module" },
+    );
+    await new Promise((res) => setTimeout(res, 3000));
+    worker_a.terminate();
+    const worker_b = new Worker(
+      new URL("fixture_b.ts", import.meta.url).href,
+      { type: "module" },
+    );
+    await new Promise((res) => setTimeout(res, 3000));
+    worker_b.terminate();
```

Comment:
Maybe write fixtures to disk in this script?

Reply:
🤦  yes that makes a huge amount of sense!